lib/paths.cf

```diff
@@ -570,6 +586,25 @@ bundle common paths
 
       "path[shadow]"       string => "/etc/security/passwd";
 
+    termux::
+      # some adjustments for the basics
+      "path[tar]"           string => "/usr/bin/tar";
+      "path[true]"          string => "/usr/bin/true";
+      "path[false]"         string => "/usr/bin/false";
+      "path[sysctl]"        string => "/usr/bin/sysctl";
+
+      # now, mangle the values by prepending the TERMUX_PREFIX
+      "files_path" string => "/data/data/com.termux/files";
+      "root_path" string => "$(root_path)$(const.dirsep)usr";
+      "tmp_paths" slist => getindices("path");
+      "tmp_path[$(tmp_paths)]" string => "$(files_path)$(path[$(tmp_paths)])";
+      "path[$(tmp_paths)]" string => "$(tmp_path[$(tmp_paths)])";
+
+    !(termux|windows)::
+      # reasonable defaults for unix systems to allow for writing
+      # more portable paths between termux and other systems
+      "root_path" string => "/";
```

Comment:
Does it make sense to have this be `/` vs having it be an empty string?

e.g.
`"$(paths.root_path)/etc/fstab"` == `//etc/fstab` vs `/etc/fstab`

Reply:
correct. I would have caught that. Thanks! :)